src/tutorial/scheduling/datetime-cycling.rst

```diff
@@ -151,33 +151,30 @@ Recurrence Formats
    As with integer cycling, recurrences start at the :term:`initial cycle
    point` by default. We can override this in two ways:
 
-By giving an arbitrary start cycle point (``datetime/recurrence``):
+.. rubric::
+   By giving an arbitrary start cycle point (``datetime/recurrence``):
 
-.. csv-table::
```

Comment:
These tables weren't working well (horizontal scrollbar and recurrences split over two lines), switched to a regular definition list.